assets/stylesheets/layouts/_header.scss

```diff
@@ -9,6 +9,12 @@
     margin-top: .625rem;
   }
 
+  // Don't show mobile menu items on the main menu.
+  [data-main-menu] [data-exclude-main],
```

Comment:
Nope. This is controlling hiding items **in the main menu**, so we have support for `[data-exclude-main]` and `[data-mobile-only]` that would control hiding elements in the main menu.

The mobile menu, on the other hand, is controlled by the shared-header package, which simply ignores elements based on the opposite data attributes (`[data-exclude-mobile]` and `[data-main-only]`).